app/views/govuk_publishing_components/components/docs/checkboxes.yml

```diff
@@ -35,6 +35,38 @@ examples:
           value: "green"
         - label: "Blue"
           value: "blue"
+  with_a_heading_on_one_checkbox:
+    description: One checkbox does not require a fieldset and therefore does not require a legend. If a heading is supplied, this will be shown with one checkbox.
```

Comment:
When I check on the component guide, this particular example is still rendered with a fieldset. Should this be removed in order to make it consistent with the description?

Reply:
Sorry, I think this is to do with the description here rather than the behaviour of the component. Will re-word.